package/suse-migration-services-spec-template

```diff
@@ -32,6 +32,9 @@ Source:           suse-migration-services.tar.gz
 BuildRoot:        %{_tmppath}/%{name}-%{version}-build
 BuildRequires:    fdupes
 BuildRequires:    python-rpm-macros
+%if 0%{?suse_version} >= 1600
+BuildRequires:    %{pythons}-poetry
```

Comment:
why is `poetry-core` not enough? according to pyproject it depends on poetry-core.

Reply:
Indeed, fixed